emission/analysis/intake/segmentation/trip_segmentation_methods/dwell_segmentation_time_filter.py

```diff
@@ -199,7 +199,7 @@ def continue_just_ended(self, idx, currPoint, filtered_points_df):
             else:
                 return False
 
-    def has_trip_ended(self, prev_point, curr_point, timeseries, last10PointsDistances, last5MinsDistances, last5MinTimes):
+    def has_trip_ended(self, prev_point, curr_point, timeseries, last10PointsDistances, last5MinsDistances, last5MinTimes, transition_df, motion_df):
```

Comment:
Ditto. if we are passing in the transition_df and motion_df, why do we also need the timeseries?

Reply:
Here, it's unused. Cleared.